pkg/pb/nodestats.proto

```diff
@@ -36,6 +36,7 @@ message DailyStorageUsageRequest {
 
 message DailyStorageUsageResponse {
     message StorageUsage {
+        int64 rollup_id = 1;
```

Comment:
Isn't timestamp sufficient for this?

Reply:
When I tested I had 170 roll up entries for 10 storagenodes just for one day. So I am not sure if storagenode can trust satellite here. But yeah, maybe it’s sufficient just to replace old value

Reply:
I agree with Egon, i dont think we need a rollup_id